pennylane/shadows/classical_shadow.py

```diff
@@ -404,42 +403,48 @@ def qnode(x):
             bitstrings, recipes = qnode(x)
             shadow = qml.ClassicalShadow(bitstrings, recipes)
 
-        >>> [shadow.entropy(wires=wires, alpha=alpha, atol=1e-10) for alpha in [1., 2., 3.]]
+        >>> [shadow.entropy(wires=wires, alpha=alpha) for alpha in [1., 2., 3.]]
         [1.5419292874423107, 1.1537924276625828, 0.9593638767763727]
 
         """
 
+        if atol is not None:
+            warnings.warn(
+                "Trying to use atol in ClassicalShadow.entropy. This feature is deprecated and has been replaced by a more accurate version that projects the density matrix estimator to the closest valid density matrix in L2 norm. You can simply remove the atol keyword in your code.",
+                DeprecationWarning,
```

Comment:
`DeprecationWarning` doesn't show up correctly for reasons that I don't quite understand. We've been generally using  `UserWarning`, and there's also a `PennylaneDeprecationWarning` now.

Reply:
```suggestion
                qml.PennyLaneDeprecationWarning,
```

These are treated as errors by our CI, so we can make sure that all instances of these warnings are removed.